src/htaccess

```diff
@@ -738,6 +738,13 @@ AddEncoding br .br
     RewriteCond %{REQUEST_FILENAME} !-d
     RewriteCond %{REQUEST_URI} !\.(json|wasm|js|css|png|jpg|map)(\.(gz|br))?$
     RewriteRule (.*) index.html [QSA,L]
+    
+    # Block access to all hidden files and directories with the exception of
+    # the visible content from within the `/.well-known/` hidden directory.
```

Comment:
Unfortunately this won't work because it's already handled by the above. Though if the file inside `well-known` has a `.json` extension it's already covered by the above rule.